Docs/ProjectSpec.md

```diff
@@ -114,6 +114,7 @@ Note that target names can also be changed by adding a `name` property to a targ
   - `none` - sorted alphabetically with all the other files
   - `top` - at the top, before files
   - `bottom` (default) - at the bottom, after other files
+- [ ] **groupsOrder**: **[[GroupOrder]](#groupOrder)** - An order of groups.
```

Comment:
Not in love with the name `groupsOrder`, but I also can't think of something better right now.

Reply:
Not only you, I don't like it too, but I couldn't find more lovely name :( I'll try to think about it again.

Reply:
How about `groupOrdering`?